Objects/Objects/Geometry/Brep.cs

```diff
@@ -306,6 +307,33 @@ internal void OnDeserialized(StreamingContext context)
             f.Brep = this;
       }
     }
+
+    /// <summary>
+    /// Returns a new transformed Brep. Note that `displayMesh` is *NOT* transformed. If you fall back on
+    /// the `displayMesh`, make sure to call `TransformTo` on the mesh object.
+    /// </summary>
+    /// <param name="transform"></param>
+    /// <returns></returns>
+    public Brep TransformTo(Transform transform)
+    {
+      return new Brep
+      {
+        provenance = provenance,
+        units = units,
+        displayMesh = displayMesh,
```

Comment:
Is there a reason why the displayMesh isn't transformed here as well?

Reply:
performance 🙃
it's really slow to transform large meshes